### DISCUSSION TITLE
Overlaid texts when exporting PDF from InDesign book

### DISCUSSION THREAD

**Discussion by patbel** [sumatrapdfreader/sumatrapdf#3002] [Q&A]
I have this PDF file attached that if opened with Sumatra shows overlaid texts.
I tried also with Adobe reader, Firefox, Edge, Chrome, none of them has the same problem.

[Example.pdf](https://github.com/sumatrapdfreader/sumatrapdf/files/9703523/Example.pdf)

**Comment by GitHubRulesOK** [comment]
Do you mean it looks like this in other viewers ALWAYS AVOID such constructs 
PDF Accessibility readers will throw hands up in horror of such textual ovelay abuses.

![image](https://user-images.githubusercontent.com/12754913/193831279-086d4656-7fc3-4a86-b2ec-efdabc50bb77.png)
![image](https://user-images.githubusercontent.com/12754913/193833533-8e9a793a-5ea9-4d26-ab50-15b08aa91804.png)
![image](https://user-images.githubusercontent.com/12754913/193833881-1bf4ad28-1067-430a-b35c-6dd58e537b75.png)


SumatraPDF should show only German same as Edge but the file should be saved one language at a time like this
[3002 Example_Optimized.pdf](https://github.com/sumatrapdfreader/sumatrapdf/files/9707382/3002.Example_Optimized.pdf)

![image](https://user-images.githubusercontent.com/12754913/193832272-e1f7811c-b0c5-4358-a396-7946bb15d487.png)
